doc/man3/flux_request_encode.adoc

```diff
@@ -16,7 +16,7 @@ SYNOPSIS
                                   const char *json_str);
 
  flux_msg_t *flux_request_encode_raw (const char *topic,
-                                      void *data, int *len);
+                                      void *data, int len);
 
```

Comment:
oh whoops, nevermind.  This is a typo fix, not a change related to the other stuff.